contracts/p1/AssetRegistry.sol

```diff
@@ -166,6 +166,25 @@ contract AssetRegistryP1 is ComponentP1, IAssetRegistry {
         }
     }
 
+    /// @inheritdoc IAssetRegistry
+    function validateCurrentAssets() external view {
```

Comment:
We should add a check here to revert elegantly if the registries are not set up. Otherwise it ends up reverting with a " fuinction returns unexpected amount of data" when trying to return the value for the first item, whent he Asset Plugin Registry address is 0 for example